packages/desktop-gui/src/specs/specs-store.js

```diff
@@ -136,8 +136,8 @@ export class SpecsStore {
     this.filter = null
   }
 
-  @action setSelectedSpec (spec) {
-    this.selectedSpec = spec
+  @action setSelectedSpecs (specs) {
```

Comment:
Maybe we can imagine more descriptive name for this. It looks like synonym for the Choosen spec method for me :)